cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -502,91 +715,76 @@
                 if (lock) {
                     this.lockToolbar = true;
                     // make button look disabled
-                    this.toolbarTrigger.css('opacity', 0.2);
+                    this.ui.toolbarTrigger.css('opacity', 0.2);
                 } else {
                     this.lockToolbar = false;
                     // make button look disabled
-                    this.toolbarTrigger.css('opacity', 1);
+                    this.ui.toolbarTrigger.css('opacity', 1);
                 }
             },
 
-            _loader: function (loader) {
-                if (loader) {
-                    this.toolbarTrigger.addClass('cms-toolbar-loader');
-                } else {
-                    this.toolbarTrigger.removeClass('cms-toolbar-loader');
-                }
-            },
-
-            _debug: function () {
+            /**
+             * Handles the debub bar when `DEBUG=true` on top of the toolbar.
+             *
+             * @method _debug
+             * @private
+             */
+            _debug: function _debug() {
                 var that = this;
                 var timeout = 1000;
                 var timer = function () {};
 
                 // bind message event
-                var debug = this.container.find('.cms-debug-bar');
-                debug.bind('mouseenter mouseleave', function (e) {
+                var debug = this.ui.container.find('.cms-debug-bar');
+                debug.on(this.mouseEnter + ' ' + this.mouseLeave, function (e) {
                     clearTimeout(timer);
 
-                    if (e.type === 'mouseenter') {
+                    if (e.type === that.mouseEnter) {
                         timer = setTimeout(function () {
-                            that.openMessage(that.config.lang.debug);
+                            that.openMessage({
+                                message: that.config.lang.debug
+                            });
                         }, timeout);
                     }
                 });
             },
 
-            _screenBlock: function () {
+            /**
+             * This shows a dark screen with a note "This page is a redirect"
+             * on a page where the settings have been modified to redirect to
+             * another page.
+             *
+             * @method _screenBlock
+             * @private
+             */
+            _screenBlock: function _screenBlock() {
+                var that = this;
                 var interval = 20;
-                var blocker = this.screenBlock;
+                var blocker = this.ui.screenBlock;
                 var sideframe = $('.cms-sideframe');
 
                 // automatically resize screenblock window according to given attributes
-                $(window).on('resize.cms.screenblock', function () {
-                    var width = $(this).width() - sideframe.width();
-
+                $(window).on(this.resize, function () {
```

Comment:
well since screen block is a position fixed element with top 0, left 0 and width/height 100% it doesn't make any sense to listen to window resize, especially since we changed the way sideframe works.

or at the very least cache `$(window)` reference